gitgoing/gitgoing.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+NEURON_COMPONENTS = ('soma', 'axon', 'dendrite', 'synapse')
+
+def mean(x):
```

Comment:
Maybe avoid overloading these function names from numpys? Something cheesy like `my_mean` and `my_std` might work well for this purpose.